dbt/adapters/athena/impl.py

```diff
@@ -833,3 +862,32 @@ def is_list(self, value: Any) -> bool:
         a list since this is complicated with purely Jinja syntax.
         """
         return isinstance(value, list)
+
+    @staticmethod
+    def _get_table_input(table: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
the type of `table` should not be a dict but a type from boto3-stubs no ? like `GetTableRef` or something like that

Reply:
and return type should be fixed as well I guess